File: modul-03/python/PRAK302-2310817310003-Mfaridsetiawan.py
nilai = int(input(' '))
 
hasil = None
if nilai > 80:
    hasil = 'A'
elif nilai <= 79 and nilai > 70:
    hasil = 'B'
elif nilai <= 69 and nilai > 60:
    hasil = 'C'
elif nilai <= 59 and nilai >= 50:
    hasil = 'D'
elif nilai < 50:
    hasil = 'E'


 
print(' {} =  {}'.format(nilai, hasil))

File: modul-03/bahasa c/PRAK302-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main()
{
    int nilai;
    char hasil;

    printf(" ");
    scanf("%d", &nilai);

    if (nilai > 80)
    {
        hasil = 'A';
    }
    else if (nilai <= 79 & nilai > 70)
    {
        hasil = 'B';
    }
    else if (nilai <= 69 & nilai > 60)
    {
        hasil = 'C';
    }
    else if (nilai <= 59 & nilai >= 50)
    {
        hasil = 'D';
    }
    else if (nilai < 50 & nilai >= 0)
    {
        hasil = 'E';
    }
    else
    {
        printf("Inputan salah\n");
        return 0;
    }

    printf(" %d = %c\n", nilai, hasil);

    return 0;
}